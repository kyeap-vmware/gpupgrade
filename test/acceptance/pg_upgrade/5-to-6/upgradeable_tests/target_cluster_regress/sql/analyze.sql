-- Test to ensure ANALYZE was executed on the target cluster during finalize.

select relname, reltuples, relpages from pg_class where relname like 'p3_sales%' order by relname;
select * from pg_stats where tablename like 'p3_sales%' order by tablename, attname;
select pgc.relname from pg_stat_last_operation pgl, pg_class pgc where pgl.objid=pgc.oid and pgc.relname like 'p3_sales%' and staactionname='ANALYZE' order by pgc.relname;
