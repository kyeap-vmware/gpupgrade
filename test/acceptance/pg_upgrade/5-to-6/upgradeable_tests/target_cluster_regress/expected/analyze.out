-- Test to ensure ANALYZE was executed on the target cluster during finalize.

select relname, reltuples, relpages from pg_class where relname like 'p3_sales%' order by relname;
 relname                                                         | reltuples | relpages 
-----------------------------------------------------------------+-----------+----------
 p3_sales                                                        | 0.0       | 0        
 p3_sales_1_prt_2                                                | 0.0       | 0        
 p3_sales_1_prt_2_2_prt_2                                        | 0.0       | 0        
 p3_sales_1_prt_2_2_prt_2_3_prt_other_regions                    | 0.0       | 1        
 p3_sales_1_prt_2_2_prt_2_3_prt_usa                              | 2.0       | 1        
 p3_sales_1_prt_2_2_prt_other_months                             | 0.0       | 0        
 p3_sales_1_prt_2_2_prt_other_months_3_prt_other_regions         | 0.0       | 1        
 p3_sales_1_prt_2_2_prt_other_months_3_prt_usa                   | 0.0       | 1        
 p3_sales_1_prt_outlying_years                                   | 0.0       | 0        
 p3_sales_1_prt_outlying_years_2_prt_2                           | 0.0       | 0        
 p3_sales_1_prt_outlying_years_2_prt_2_3_prt_other_regions       | 0.0       | 1        
 p3_sales_1_prt_outlying_years_2_prt_2_3_prt_usa                 | 0.0       | 1        
 p3_sales_1_prt_outlying_years_2_prt_other_m_3_prt_other_regions | 0.0       | 1        
 p3_sales_1_prt_outlying_years_2_prt_other_months                | 0.0       | 0        
 p3_sales_1_prt_outlying_years_2_prt_other_months_3_prt_usa      | 0.0       | 1        
(15 rows)
select * from pg_stats where tablename like 'p3_sales%' order by tablename, attname;
 schemaname | tablename                          | attname | inherited | null_frac | avg_width | n_distinct | most_common_vals | most_common_freqs | histogram_bounds | correlation | most_common_elems | most_common_elem_freqs | elem_count_histogram 
------------+------------------------------------+---------+-----------+-----------+-----------+------------+------------------+-------------------+------------------+-------------+-------------------+------------------------+----------------------
 public     | p3_sales                           | day     | t         | 0.0       | 4         | -0.5       | {20}             | {1}               |                  |             |                   |                        |                      
 public     | p3_sales                           | id      | t         | 0.0       | 4         | -0.5       | {1}              | {1}               |                  |             |                   |                        |                      
 public     | p3_sales                           | month   | t         | 0.0       | 4         | -0.5       | {1}              | {1}               |                  |             |                   |                        |                      
 public     | p3_sales                           | region  | t         | 0.0       | 4         | -0.5       | {usa}            | {1}               |                  |             |                   |                        |                      
 public     | p3_sales                           | year    | t         | 0.0       | 4         | -0.5       | {2002}           | {1}               |                  |             |                   |                        |                      
 public     | p3_sales_1_prt_2_2_prt_2_3_prt_usa | day     | f         | 0.0       | 4         | -0.5       | {20}             | {1}               |                  | 1.0         |                   |                        |                      
 public     | p3_sales_1_prt_2_2_prt_2_3_prt_usa | id      | f         | 0.0       | 4         | -0.5       | {1}              | {1}               |                  | 1.0         |                   |                        |                      
 public     | p3_sales_1_prt_2_2_prt_2_3_prt_usa | month   | f         | 0.0       | 4         | -0.5       | {1}              | {1}               |                  | 1.0         |                   |                        |                      
 public     | p3_sales_1_prt_2_2_prt_2_3_prt_usa | region  | f         | 0.0       | 4         | -0.5       | {usa}            | {1}               |                  | 1.0         |                   |                        |                      
 public     | p3_sales_1_prt_2_2_prt_2_3_prt_usa | year    | f         | 0.0       | 4         | -0.5       | {2002}           | {1}               |                  | 1.0         |                   |                        |                      
(10 rows)
select pgc.relname from pg_stat_last_operation pgl, pg_class pgc where pgl.objid=pgc.oid and pgc.relname like 'p3_sales%' and staactionname='ANALYZE' order by pgc.relname;
 relname                                                         
-----------------------------------------------------------------
 p3_sales                                                        
 p3_sales_1_prt_2_2_prt_2_3_prt_other_regions                    
 p3_sales_1_prt_2_2_prt_2_3_prt_usa                              
 p3_sales_1_prt_2_2_prt_other_months_3_prt_other_regions         
 p3_sales_1_prt_2_2_prt_other_months_3_prt_usa                   
 p3_sales_1_prt_outlying_years_2_prt_2_3_prt_other_regions       
 p3_sales_1_prt_outlying_years_2_prt_2_3_prt_usa                 
 p3_sales_1_prt_outlying_years_2_prt_other_m_3_prt_other_regions 
 p3_sales_1_prt_outlying_years_2_prt_other_months_3_prt_usa      
(9 rows)
